### DISCUSSION TITLE
Where can I find the interface to install and update packages?

### DISCUSSION THREAD

**Discussion by romu545** [posit-dev/positron#4409] [Q&A]
![Captura de pantalla 2024-08-20 100300](https://github.com/user-attachments/assets/a83d9ac8-c10c-4e78-98dc-ecdfd88ee4f6)

**Comment by anbrav0** [comment]
I brought up a somewhat similar discussion point here #4090 regarding improvements for package management wizard, they also talked about some here #1369 and here #1391

**Reply by juliasilge** [reply]
Thanks for chiming in @anbrav0! 🙌 

For now @romu545 we do not have a GUI / wizard option in Positron to install or update packages. Instead you will want to use `install.packages()` and `update.packages()`.

**Reply by Vancamjr** [reply]
Correction:  Instead you will have to use install.packages() and update.packages()

I like the package manager in RStudio and was excited to see how it was implemented in positron for multiple languages.  I’m disappointed to see you take the easy path of omitting this valued functionality.

**Reply by juliasilge** [reply]
We have plans to work on an environments/package manager outlined in #1391, if you'd like to add additional info on your use case.

**Reply by Steinthal** [reply]
As mentioned in #1369, there's an extension [Positron R Package Manager](https://github.com/kv9898/positron-r-package-manager) that might help in the meantime.
